src/session_impl.cpp

```diff
@@ -5521,11 +5521,11 @@ namespace aux {
 		if (m_outstanding_router_lookups > 0) return;
 
 		m_dht_storage = m_dht_storage_constructor(m_dht_settings);
-		m_dht = boost::make_shared<dht::dht_tracker>(
+		m_dht = std::make_shared<dht::dht_tracker>(
 			static_cast<dht_observer*>(this)
 			, std::ref(m_io_service)
 			, std::bind(&session_impl::send_udp_packet, this, false, _1, _2, _3, _4)
-			, boost::cref(m_dht_settings)
+			, std::cref(m_dht_settings)
```

Comment:
I don't think the reference wrappers are necessary anymore, with perfect forwarding in C++11